internal/test/builder/infrastructure.go

```diff
@@ -35,6 +35,12 @@ var (
 	// GenericInfrastructureMachineTemplateCRD is a generic infrastructure machine template CRD.
 	GenericInfrastructureMachineTemplateCRD = untypedCRD(InfrastructureGroupVersion.WithKind(GenericInfrastructureMachineTemplateKind))
 
+	// GenericInfrastructureMachinePoolTemplateKind is the Kind for the GenericInfrastructureMachinePoolTemplate.
+	GenericInfrastructureMachinePoolTemplateKind = "GenericInfrastructureMachinePoolTemplate"
+
+	// GenericInfrastructureMachinePoolObjectKind is the Kind for the GenericInfrastructureMachinePoolObject.
+	GenericInfrastructureMachinePoolObjectKind = "GenericInfrastructureMachinePoolObject"
```

Comment:
```suggestion
	// GenericInfrastructureMachinePoolKind is the Kind for the GenericInfrastructureMachinePool.
	GenericInfrastructureMachinePoolKind = "GenericInfrastructureMachinePool"
```
I think we don't need the Object suffix

Reply:
Same for TestInfrastructureMachinePoolObjectKind